FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Metrics/ProcessLatencyMetric.cs

```diff
@@ -10,9 +10,9 @@ namespace Microsoft.Health.Fhir.Synapse.Common.Metrics
     public class ProcessLatencyMetric : DiagnosticMetrics
     {
         public ProcessLatencyMetric()
-            : base("ResourceLatency", MetricsCategories.Latency, new Dictionary<string, object>
+            : base("ProcessLatency", MetricsCategories.Latency, new Dictionary<string, object>
```

Comment:
maybe a const string for it

Reply:
Maybe we can refine it later, since all metrics use hard code string for 'name' dimension. 
Since this string is only used here, maybe const is not quite necessary for now.

Reply:
Added const for all metric's name.